src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -25,6 +24,24 @@ public static Line of(List<Point> points) {
         return new Line(points);
     }
 
+    public int nextHorizontalNumber(int horizontalNumber) {
+        if (pointOnTheLeft(horizontalNumber)) {
+            return --horizontalNumber;
+        }
+        if (pointOnTheRight(horizontalNumber)) {
+            return ++horizontalNumber;
+        }
```

Comment:
이렇게 축약형으로 표현하기 보단, 명확하게 다음과 같이 작성해주는게 어떨까요?
```suggestion
        if (pointOnTheLeft(horizontalNumber)) {
            horizontalNumber -= 1;
            return horizontalNumber;
        }
        if (pointOnTheRight(horizontalNumber)) {
            horizontalNumber += 1;
            return horizontalNumber;
        }
```
그리고 이렇게 표현하기보단 차라리 다음과 같은 형태로 작성해주면 더 좋을 것 같아요!
```suggestion
        horizontalNumber += /* 임의의 메소드 */;
```